src/audio/CMakeLists.txt

```diff
@@ -73,11 +81,15 @@ check_optimization(fma -mfma -DOPS_FMA)
 check_optimization(hifi2ep -mhifi2ep -DOPS_HIFI2EP)
 check_optimization(hifi3 -mhifi3 -DOPS_HIFI3)
 
-set(sof_audio_modules volume src)
+if(CONFIG_SRC)
```

Comment:
Host/testbench build doesn't have Kconfig atm, so shouldn't rely on CONFIG_* values because they will be undefined always

Reply:
Removed